src/database/seeders/20190708175931-article.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
+  up: queryInterface =>
```

Comment:
Hello, I do not think you need this seeders, Is there any reason why you have them here?

Reply:
Thank you for this review. 
The reason for this seeding is for the database to have data in the database before the tests are run.
This is because I noticed there is a `pretest` hook that is called when running the `test` script